Runtime/Plugins/GameLiftServerSDKNet45.dll.config

```diff
@@ -0,0 +1,3 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as above - clarifying if we generally want to include this?

Reply:
Seeing multiple files of this sort - adding all types I could identify. We may want to double check if these are required to be checked in to avoid bloating up the package
`.meta`
`.dll.config`
`.dll.config.meta`
`.dll.meta`
`.pdb`
`.pdb.meta`

Reply:
dll.config and pdb will require changes to the downloaded sdk csproj file, going to fix that in a later PR.